repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -128,9 +128,7 @@ class ReposeJettyServer(val clusterId: String,
           }
         }
 
-        Option(ssl.isTlsRenegotiationAllowed).foreach { tls =>
-          cf.setRenegotiationAllowed(tls)
-        }
+        cf.setRenegotiationAllowed(Option(ssl.isTlsRenegotiationAllowed).getOrElse[java.lang.Boolean](false))
```

Comment:
Ew, switching from boxing to a Scala Boolean to boxing to a Java Boolean. The type system differences are super apparent in this line. Could you use the Java `Boolean.FALSE` constant instead (https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#FALSE)?

I never realized that XJC boxes booleans.